tests/plugins/imagebase/imagebase.js

```diff
@@ -0,0 +1,59 @@
+/* bender-tags: editor,widget */
+/* bender-ckeditor-plugins: imagebase,toolbar */
+
+( function() {
+	'use strict';
+
+	function assertUpcast( bot, data, widget ) {
```

Comment:
You can get rid of it and reuse widget just like I did in 08ef0dcf6a88a99e79b55ef46e361f73c192b844.

Reply:
I don't see any similar method in widget tools for that – for now I'd stick to my custom `assertUpcast`.

Reply:
Look at the commit I have referneced:

```js
'test easyimage_class - changed': function() {
	widgetTestsTools.assertWidget( {
		count: 1,
		widgetOffset: 0,
		nameCreated: 'easyimage',
		html: CKEDITOR.document.getById( 'changedClass' ).getHtml(),
		bot: this.editorBots.classCustomized
	} );
},
```

It does check widget count and a widget type, abstracting away how to get widget count etc.